TechTable/tt_table.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Courses offered at IIIT</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+ </head>
+ <body>
+  <div class="container">
+   <div class="table-responsive">
+    <h1 align="center">Courses offered at IIIT</h1>
+    <br />
+    <div align="center">
+     <button type="button" name="load_courses" id="load_courses" class="btn btn-info">Load Courses</button>
+    </div>
+    <br />
+    <div id="courses_table">
+    </div>
+   </div>
+  </div>
+ </body>
+</html>
+
+<script>
+$(document).ready(function(){
+ $('#load_courses').click(function(){
+  $.ajax({

```

Comment:
Again, why not use the more succinct `$.get` here? https://api.jquery.com/jquery.get/

Also I'm not sure why there are two files displaying table. Is one of them an older file which isn't used anymore? 

If so, make a separate folder `misc` or `old` and move these files there.

Reply:
The file [tt_table.html](https://github.com/ruhmamehek/tech-tree/blob/master/TechTable/Old/tt_table.html) was the older version for the Course Table, and has been moved to a separate [folder](https://github.com/ruhmamehek/tech-tree/tree/master/TechTable/Old).